cmd/kwil-admin/cmds/migration/network_status.go

```diff
@@ -0,0 +1,72 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/kwilteam/kwil-db/cmd/common/display"
+	"github.com/kwilteam/kwil-db/cmd/kwil-admin/cmds/common"
+	"github.com/kwilteam/kwil-db/core/types"
+	"github.com/spf13/cobra"
+)
+
+func networkStatusCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "network-status",
+		Short: "Get the migration status of the network.",
+		Example: `# Get the migration status of the network.
```

Comment:
I think the description should say what network it's supposed to work on (old or new?)

Reply:
BTW, this particular command does make sense on both networks.  Does/can it?

Reply:
Yeah, it can be used on both the networks.